corehq/apps/cloudcare/static/cloudcare/js/debugger/debugger.js

```diff
@@ -436,6 +496,17 @@ hqDefine('cloudcare/js/debugger/debugger', function () {
                 }
             },
         };
+        ko.bindingHandlers.clipboardButton = {
+            init: function(element, valueAccessor) {
+                new Clipboard('.clickable',
```

Comment:
should this enable clipboard on the element that was clicked? I think this will enable it for all elements with the `.clickable` class.